cmake/faceliftMacros.cmake

```diff
@@ -136,67 +136,98 @@ function(facelift_generate_code )
     set(multiValueArgs IMPORT_FOLDERS)
     cmake_parse_arguments(ARGUMENT "${options}" "${oneValueArgs}" "${multiValueArgs}" ${ARGN})
 
+    # generate only interfaces which has changed since last build
+    file(GLOB_RECURSE QFACE_FILES ${ARGUMENT_INTERFACE_DEFINITION_FOLDER}/*.qface)
```

Comment:
you also need to check if the code generator itself has changed.

Reply:
Fixed: line 167